docs/modules/ROOT/pages/explanations/vshn-postgres.adoc

```diff
@@ -0,0 +1,10 @@
+= VSHN Postgres Functions
+
+The set of functions applied to a VSHN Postgres composition.
+
+== Function URL
+
+The function URL creates a new `POSTGRES_URL` entry in _Data_ of a connection secret. The secret is wrapped around a kube object. The value is defined as `postgres://user:password@host:port/db`. Once it is executed the client has access to the URL of its database via connection secret.
```

Comment:
Nit: we don't write into the intermediate connection secret anymore, so this description could be a bit confusing.

Reply:
yes this has to be updated